cmd/setup.go

```diff
@@ -559,14 +559,6 @@ func configureInflux(conf *globalconfig.Influx) (*server.Influx, error) {
 		return nil, nil
 	}
 
-	// TODO remove yaml file
```

Comment:
brauchen wir das nicht mehr weil in migrate?

Reply:
Jep